### ISSUE TITLE
Support for modifier key actions.

### ISSUE THREAD

**Issue by LiamKarlMitchell** [ryanmcnz/gmweb#2]
```Ctrl```+```Z```, ```Ctrl```+```X```, ```Ctrl```+```C```, ```Ctrl```+```V``` etc.

**Comment by ryanmcnz** [comment]
Yup, also things like CTRL + arrow, CTRL + SHIFT + arrow, etc... when in text areas.
